src/plugins/version/index.js

```diff
@@ -0,0 +1,11 @@
+class Version {
+    constructor(config) {
+        config.on('command:version', this.handleCommand.bind(this, config.version));
+    }
+
+    handleCommand(version) {
+        console.log('Kosmtik version', version);
```

Comment:
Maybe this plugin could iterate over every "external" plugin and display their versions too ?

Reply:
I looked into it. That could work, but would require hacking `require` paths to get the `package.json` that contains the version. 

I think it would be preferable if the plugin could have a field that contains the version (just like the `Config` instance here). But none of the plugins have that field currently, and it would require duplicating the plugin instantiation logic in `Config` to instantiate each instance, purely for reading the version.